### PR TITLE
Fix --r breaking reminder

### PR DESCRIPTION
**Pull request by blimyj**
Fix issue #268

### CODE REVIEW

**Comment by blimyj** [comment]
Fix issue #212

File: src/main/java/seedu/elisa/model/ItemModelManager.java [hunk 1]
```diff
@@ -363,18 +358,62 @@ public void replaceItem(Item item, Item newItem) {
...
+            }
+        }
+
+        System.out.println(index);
+        if (index >= 0) {
+            if (visualList.belongToList(newItem)) {
+                visualList.setItem(index, newItem);
+            } else {
+                visualList.remove(index);
+            }
```
Comment (sianghwee) [inline src/main/java/seedu/elisa/model/ItemModelManager.java]: You cant just copy and paste this block of code actually, cause if you look at the individual loop, the index of the item is actually getting updated at each round. So before this block you need: `index = visualList.indexOf(item)`

Comment (blimyj) [reply src/main/java/seedu/elisa/model/ItemModelManager.java]: Nice catch! I edited this as well as the initialization of the index to match the first check.

File: src/main/java/seedu/elisa/model/ItemModelManager.java [hunk 2]
```diff
@@ -363,18 +358,62 @@ public void replaceItem(Item item, Item newItem) {
...
             } else {
                 reminderList.remove(index);
             }
 
-            //Old reminder must be in active since it already rang.
-            //Find the old reminder from activeReminders and remove it.
-            activeReminders.remove(item);
-            //Put the new reminder in futureReminders if it will be rung later.
-            futureReminders.add(newItem);
+            System.out.println("Help");
```
[the remark this answers is not in the document]
Comment (blimyj) [reply src/main/java/seedu/elisa/model/ItemModelManager.java]: Resolved.

File: src/main/java/seedu/elisa/model/ItemModelManager.java [hunk 3]
```diff
@@ -363,18 +358,62 @@ public void replaceItem(Item item, Item newItem) {
             }
         }
 
+        System.out.println("Boo");
+        System.out.println(reminderList.indexOf(item));
```
[the remark this answers is not in the document]
Comment (blimyj) [reply src/main/java/seedu/elisa/model/ItemModelManager.java]: Woops! My bad indeed, resolved.

**Pull request merged**